### DISCUSSION TITLE
How to translate "list" and "tuple"

### DISCUSSION THREAD

**Discussion by adrianliaw** [python/python-docs-zh-tw#401] [General]
Marr 提出了一個[翻譯「List」和「Tuple」的觀點](https://medium.com/@l34marr/list-tuple-translation-872c3d449887)，我認同「串列」和「值組」這兩個翻譯，沒什麼問題的話可以加入 glossary 裡。

**Comment by StevenHsuYL** [comment]
我之後也會使用「值組」作為 tuple 的翻譯。
我目前正在處理 glossary.po。

**Comment by scan33scan33** [comment]
您好 我最近在審定一本書，也在想tuple中文要如何翻譯。
看到這個thread。
想請問「值組」是你們建議的翻譯嗎？

**Comment by StevenHsuYL** [comment]
在市面上或學術界沒有共通的翻譯的情況下，我覺得在此訂下初步的共識並持續推廣，是沒有問題的。
我認同 @adrianliaw 提出的「值組」的翻譯，在 glossary 的翻譯草稿中，也會使用此翻譯。
也希望我們能逐步建立繁體中文的共用術語，使國內讀者有共同的溝通詞彙。

**Comment by scan33scan33** [comment]
好唷，我其實也覺得「值組」可以。
如果有任何改變，可以再update這個issue。

BTW, 我一般來講都是看這個: https://terms.naer.edu.tw/search/?q=%22tuple%22&field=ti&num=50  去查可能翻譯。

感謝您

**Comment by StevenHsuYL** [comment]
我在翻譯時，第一優先也是參考教育部辭典。但不太確定時，我會使用 google 稍微瀏覽繁體中文學術網頁常用的翻譯，以及繁體中文 wiki 是否與教育部辭典一致，如果翻譯有差異，可能還會參考 Microsoft Docs 中文版 ( https://docs.microsoft.com/zh-tw/ )，最後綜合這些結果來做出最後的決定。

不過我覺得最好的方法，還是大家在這邊互相討論產生共識，這樣未來有翻譯疑義時也可以回溯討論串並提供建議。

謝謝！

**Comment by hwhsu1231** [comment]
誠如 Marr 那篇文章提到的：

> List 相對又簡單一點, 有人譯成 “清單” 有人譯成 “列表”, 都勉強說得過去...

關於「list」的翻譯，我的確也看到有人翻譯為「清單/列表」。

不過我傾向於將「Manifest」翻譯為「清單」。也就是說：

* Manifest: 清單
* List: 列表

我目前是習慣這麼翻譯的。

參考

* [清單檔案 - 維基百科](https://zh.wikipedia.org/zh-tw/%E6%B8%85%E5%8D%95%E6%96%87%E4%BB%B6)

**Reply by liying-c** [reply]
看了一下目前的repository，現在tuple使用「元組」一詞的翻譯遠大於使用「值組」，是否應統一並加入repository的文件？
<img width="316" alt="image" src="https://github.com/python/python-docs-zh-tw/assets/67285884/3980a62b-65fc-4a5f-933e-47a873fc5fdf">

<img width="311" alt="image" src="https://github.com/python/python-docs-zh-tw/assets/67285884/a3b1f9c2-8745-42b3-a57b-e0511046b94f">

**Reply by StevenHsuYL** [reply]
感謝提議。
目前 tuple 的繁體中文翻譯仍維持「元組」不變（例如：[術語表](https://docs.python.org/zh-tw/3.12/glossary.html?highlight=%E5%85%83%E7%B5%84#term-immutable)），如果有更好的提議歡迎隨時討論。
謝謝！
